packages/data-point/lib/entity-types/entity-request/resolve.test.js

```diff
@@ -401,7 +400,6 @@ describe('resolve', () => {
     return transform('request:a9', {}).catch(err => {
       expect(err.statusCode).toEqual(404)
       expect(err.message).toMatchSnapshot()
-      expect(console.info).toBeCalled()
```

Comment:
we're using jest to mock `console.info`, so we can remove some of that code as well. please search the file for `console.info` and remove any code that you find if it's no longer being used.